face/README.md

```diff
@@ -25,11 +25,12 @@ Explore our samples and discover the things you can build with face service.
 > [!NOTE]
 > The [face liveness detection solution](https://learn.microsoft.com/azure/ai-services/computer-vision/tutorials/liveness) requires additional mobile/web SDKs to start the camera, guide the end-user in adjusting their position, and compose the liveness payload. See the frontend app integration samples:
 >
-> | Platform | Sample | Dependency (Client SDK) |
+> | Platform | Sample | Dependency (Client SDK API Reference) |
 > | :- | :- | :- |
-> | Android | [azure-ai-vision-sdk/tree/main/samples/kotlin/face/FaceAnalyzerSample](https://github.com/Azure-Samples/azure-ai-vision-sdk/tree/main/samples/kotlin/face/FaceAnalyzerSample) | [azure-ai-vision-faceanalyzer](https://azure.github.io/azure-sdk-for-android/azure-ai-vision-faceanalyzer/com/azure/android/ai/vision/faceanalyzer/package-summary.html) |
-> | iOS | [azure-ai-vision-sdk/tree/main/samples/swift/face/FaceAnalyzerSample](https://github.com/Azure-Samples/azure-ai-vision-sdk/tree/main/samples/swift/face/FaceAnalyzerSample) | [AzureAIVisionFace](https://azure.github.io/azure-sdk-for-ios/AzureAIVisionFace/index.html) |
-> | Web | [azure-ai-vision-sdk/tree/main/samples/web](https://github.com/Azure-Samples/azure-ai-vision-sdk/tree/main/samples/web) | azure-ai-vision-faceanalyzer |
+> | Android | [FaceLivenessDetectorSample-Android](https://aka.ms/azure-ai-vision-face-liveness-client-sdk-android-readme) | [API-Reference](
+https://aka.ms/azure-ai-vision-face-liveness-client-sdk-android-api-reference) |
+> | iOS | [FaceLivenessDetectorSample-iOS](https://aka.ms/azure-ai-vision-face-liveness-client-sdk-ios-readme) | [API-Reference](https://aka.ms/azure-ai-vision-face-liveness-client-sdk-ios-api-reference) |
+> | Web | [FaceLivenessDetectorSample-Web](https://aka.ms/azure-ai-vision-face-liveness-client-sdk-web-readme) | [API-Reference](https://aka.ms/azure-ai-vision-face-liveness-client-sdk-web-api-reference) |
```

Comment:
Regarding "API-Reference", do we have a more informative text to show here? (e.g., the library name, or the entry class name)